app/javascript/dashboard/components/widgets/conversation/components/GalleryView.vue

```diff
@@ -46,9 +46,23 @@
           </span>
         </div>
         <div
-          class="items-center flex gap-2 justify-end min-w-[15rem]"
+          class="items-center flex gap-2 justify-end min-w-[8rem] sm:min-w-[15rem]"
```

Comment:
I've added a responsive breakpoint, so on mobile versions all the buttons show without overflowing outside the parent.